winpr/tools/makecert/makecert.c

```diff
@@ -61,7 +61,7 @@ struct _MAKECERT_CONTEXT
 	int duration_months;
 };
 
-COMMAND_LINE_ARGUMENT_A args[] =
+static COMMAND_LINE_ARGUMENT_A args[] =
```

Comment:
@bmiklautz If compiling as single binary there are 2 ```args``` arrays globally visible. One in makecert, one in ```client/common```. Making them static resolves that issue.